README.md

```diff
@@ -10,15 +10,15 @@ get this awesome t-shirt:
 
 [![Join the chat at https://gitter.im/scalacenter/sprees](https://badges.gitter.im/scalacenter/sprees.svg)](https://gitter.im/scalacenter/sprees?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-## Scala Wave 2018 (Gdańsk, Poland) September 9th
+## Scala.IO 2018 (Lyon, France) October 30th and 31st
 
 |                         |                                                                                            |
 | ----------------------- | ------------------------------------------------------------------------------------------ |
-| Time                    | September 9th, 9:00am - 4:00pm                                                               |
-| Location                | Olivia Business Centre, Olivia FOUR, [aleja Grunwaldzka 472a](https://www.google.com/maps/search/?api=1&query=aleja+Grunwaldzka+472a%2C+Gdansk%2C+pl)					       |
-| Conference registration | **Not Required**                                                                           |
-| Event Registration      | **Free, required**: [event-registration](https://www.meetup.com/FunctionalTricity/events/253573703/)                                                  |
-| Food                    | **Free**: Sponsored by Scalac                                                          |
+| Time                    | October 30th, 9:00am - 5:00pm, October 31st 9:00am - 4:00pm                                |
+| Location                | [CPE Lyon](https://www.google.com/maps/search/?api=1&query=aleja+Grunwaldzka+472a%2C+Gdansk%2C+pl) Bâtiment Hubert Curien, Domaine Scientifique de la Doua, 43 Boulevard du 11 Novembre 1918, 69100 Villeurbanne					       |
+| Conference registration | **Required** [event registration](https://scala.io)                                        |
```

Comment:
I wouldn't call this link `event registration` given that it says below that the event registration is not required.

Reply:
no the registration to the conference is required to have access to the spree, but no additional registration is needed beyond that

Reply:
@vil1 to make it clear: it is limited to the conference participants; for the ones that would like to join only the spree, they need to pay the ticket?
Usually, the sprees are free, sponsored by a company or two... before I go ahead and promote this I need to know, thanks

Reply:
@darjutak I confirm, a ticket to the conference will be mandatory.